test/schemas.ts

```diff
@@ -2303,14 +2302,7 @@ export const v131Schema = {
         }
       }
     ),
-    activeDapp: v130Schema.dapps.activeDapp
-      ? {
-          ...v130Schema.dapps.activeDapp,
-          categories: v130Schema.dapps.activeDapp.categories ?? [
-            v130Schema.dapps.activeDapp.categoryId,
-          ],
-        }
-      : null,
+    activeDapp: null,
```

Comment:
i had typescript errors trying to define the new schema, it kept complaining that `points` did not exist on the v211Schema. i'm sure that the issue is not with how i defined the new v212Schema, but i'm not sure exactly where in this waterfall object the type definitions became messed up. i tried to remove the `any` casts further up the waterfall which seems to have done the trick. the migration here for `activeDapp` is actually not needed because this should be a session level state anyway.